docs/getting-started/zowe_v2_faq.md

```diff
@@ -6,4 +6,30 @@ The official date is TBD, the target is April 25, 2022; look for the official an
 
 ## Where can I find the current (V1) and new (V2) LTS conformancee criteria? 
 
-The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
\ No newline at end of file
+The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
+
+## Whats the difference between "server.json" and "zowe.yaml"?
```

Comment:
You should specifically call zowe.yaml "Zowe configuration file" but you can make references to zowe.yaml as the default name (or example-zowe.yaml as it is in GitHub). We replaced usages of zowe.yaml w/ this because zowe.yaml is just a name